packages/application-components/package.json

```diff
@@ -65,7 +65,7 @@
     "storybook-readme": "5.0.3"
   },
   "peerDependencies": {
-    "@commercetools-frontend/ui-kit": "9.x",
+    "@commercetools-frontend/ui-kit": "^9.2",
```

Comment:
I don't think this will help much to be honest. Usually you only get warnings when you install an unmet version of a peer dependency, which you are probably going to ignore because your console is full of them.

I would rather just mention in the appkit changelog release that version x should require a ui-kit version >= x.

Reply:
...but warnings are better than nothing? Maybe we can just do both? Given then this would even be a breaking change in strict semver.

Reply:
Sure, we can do both I guess. Good point!

Reply:
So should I merge this then 🤔 ?